packages/fxa-settings/src/pages/Signin/SigninBounced/index.stories.tsx

```diff
@@ -3,30 +3,33 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React from 'react';
-import SigninBounced, { SigninBouncedProps } from '.';
-import AppLayout from '../../../components/AppLayout';
+import SigninBounced from '.';
 import { LocationProvider } from '@reach/router';
 import { Meta } from '@storybook/react';
 import { withLocalization } from '../../../../.storybook/decorators';
+import { MOCK_ACCOUNT } from '../../../models/mocks';
 
 export default {
   title: 'Pages/Signin/SigninBounced',
   component: SigninBounced,
   decorators: [withLocalization],
 } as Meta;
 
-const ComponentWithRouter = (props: SigninBouncedProps) => (
+export const Default = () => (
   <LocationProvider>
-    <AppLayout>
-      <SigninBounced email={props.email} canGoBack={props.canGoBack} />
-    </AppLayout>
+    <SigninBounced
+      emailLookupComplete={true}
+      email={MOCK_ACCOUNT.primaryEmail.email}
+    />
   </LocationProvider>
 );
 
-export const Default = () => (
-  <ComponentWithRouter email={'example@domain.com'} />
-);
-
 export const CanGoBack = () => (
-  <ComponentWithRouter email={'example@domain.com'} canGoBack={true} />
+  <LocationProvider>
+    <SigninBounced
+      canGoBack={true}
+      emailLookupComplete={true}
```

Comment:
nit: same here re: `{true}` boolean props (although this was already using `{true}` before), per https://arcticicestudio.github.io/styleguide-javascript/rules/react/props.html#boolean-attributes-notation

Reply:
fixed!